src/arrayMethodPop.js

```diff
@@ -6,7 +6,11 @@
  */
 function applyCustomPop() {
   [].__proto__.pop2 = function() {
-    // write code here
+    if (this.length === 0) {
+      return undefined;
+    }
+    this.length = this.length - 1;
+    return this.length;
```

Comment:
pop should return removed element not the length

Reply:
U-u-ups)...